beets/ui/commands.py

```diff
@@ -39,6 +42,7 @@
 from beets.util.confit import _package_path
 
 VARIOUS_ARTISTS = u'Various Artists'
+ExtraChoice = namedtuple('ExtraChoice', ['plugin', 'id', 'long', 'callback'])
```

Comment:
Absolutely, the same thought did cross my mind while revising the code (in particular, the `if`s that compare the `input_options()` return value with raw characters struck me as a bit out of place), and paving the way for handling all choices in the same way regardless of their origin sounds great.